docs/features/role-based-access-control/development-plan.md

```diff
@@ -0,0 +1,22 @@
+# Role base access control - development plan
+
+This document describes the initial plan for the implementation of Role based access control for Marxan.
+
+## Release 1 (early December 2021)
```

Comment:
so, it assumes we will manually edit roles for the platform?

also: for the demo, it will only be effective for project, while scenario/solutions will still be "public"?

Reply:
Hum, that's a very valid point, we should have some way to edit this - I'll include CRUD actions for `projects-acl` in the scope of the initial release, and add CRUD actions for the remaining resources in the following release :+1:

> it will only be effective for project, while scenario/solutions will still be "public"?

Yes, that is the plan!